src/main/java/org/apache/sling/jcr/contentloader/package-info.java

```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 
-@org.osgi.annotation.versioning.Version("0.4.0")
+@org.osgi.annotation.versioning.Version("0.5.0")
```

Comment:
Should we increase to version 1.0?

Reply:
this would unnecessarily break compatibility with other bundles depending on this package (although probably not much bundles uses this, but who knows).
would have been good to start with 1.0 in the first place.

Reply:
Baselining does not work if major version is 0 (https://github.com/bndtools/bnd/issues/2452).

Reply:
Let's discuss separately. I'll merge as is.